test/usecases/global/references.json

```diff
@@ -0,0 +1,3 @@
+[
+    { "output_reldir": "neuron", "nmodl_flags": ["--neuron"]}
```

Comment:
hoping we will need to care about only a finite number of combinations! (i.e. will remove the need of various combinations)